test/specs/bar.spec.js

```diff
@@ -176,6 +176,44 @@ define(['d3', 'bar', 'barChartDataBuilder'], function(d3, chart, dataBuilder) {
                     expect(actualOffset).toBe(expectedOffset);
                 });
             });
+
+            // describe('Locale', () => {
```

Comment:
Nit: you can just add an 'x' as in 'xdescribe' to comment the whole thing.

Reply:
True, but we agreed on pushing a commented out test for you to have a look at, right ?

Reply:
Yeah, I mean that adding the 'x' is how you comment the test. Not a big deal though.

Reply:
Fixed